src/client/contextChannels.ts

```diff
@@ -321,34 +341,28 @@ export abstract class ChannelBase {
  * We do not intend to support creation of 'user' channels at runtime, as this would add considerable complexity when
  * implementing a channel selector component, as it would need to support a dynamic channel list
  */
-export class DesktopChannel extends ChannelBase {
-    public readonly type!: 'desktop';
-
-    /**
-     * A user-readable name for this channel, e.g: `"Red"`
-     */
-    public readonly name: string;
+export class SystemChannel extends ChannelBase {
+    public readonly type!: 'system';
 
     /**
-     * The color that should be associated within this channel when displaying this channel in a UI, e.g: `0xFF0000`.
+     * SystemChannels may well be selectable by users. Here are the hints on how to see them.
```

Comment:
> may well be selectable by users

Can we word this a little more strongly? This is the main intention of these types of channels.